components/SponsorTable.tsx

```diff
@@ -22,6 +22,9 @@ const SponsorTable = ({isVisible} : TSponsorTable) => {
                     <a href="https://www.wearedevelopers.com/" target="_blank" rel="noreferrer">
                     <Image width={150} height={150} src="/WeAreDevelopers_logo.svg" alt="WeAreDevelopers" />
                     </a>
+                </div>
+                <h3 className="sponsors_tier">Community</h3>
```

Comment:
```suggestion
                <h3 className="sponsors_tier">Media & Community</h3>
```

Reply:
Don't undestand this suggestion, I separated the two partnerships